403 action recognition demo webcam (#376)
openvinotoolkit/openvino_notebooks: notebooks/403-action-recognition-webcam/README.md

> # Live Human Action Recognition with OpenVINO

Reviewer: ```suggestion
# Live Action Recognition with OpenVINO
```
This is my personal suggestion, don't feel like you need to accept it. "Live Human" sounds strange when I said it out loud. I suggest not putting the two words together. Open to alternatives.

<p align="center" width="100%">
    <img width="50%" src="https://user-images.githubusercontent.com/10940214/148401074-6f642c53-419d-43a1-8f19-b8bdc7e3a65c.gif"> 
</p>

Human action recognition finds actions over a video. The list of actions is extensive (400 in total) and cover *Person Actions*, (e.g., drawing, drinking, laughing), *Person-Person Actions* (e.g., hugging, shaking hands), and *Person-Object Actions* (opening present, mowing the lawn, playing "instrument"). You could find several parent-child groupings on the label's list, such as braiding hair and brushing hair, salsa dancing, robot dancing, or playing violin and playing guitar. For more information about labels and dataset, see ["The Kinetics Human Action Video Dataset"](https://arxiv.org/pdf/1705.06950.pdf)

## Notebook Contents

> This notebook demonstrates live human action recognition with OpenVINO. We use the [Action Recognition Models](https://docs.openvino.ai/2020.2/usergroup13.html) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo), specifically the [Encoder](https://docs.openvino.ai/2020.2/_models_intel_action_recognition_0001_encoder_description_action_recognition_0001_encoder.html) and [Decoder](https://docs.openvino.ai/2020.2/_models_intel_action_recognition_0001_decoder_description_action_recognition_0001_decoder.html). Both models create a sequence to sequence (`"seq2seq"`)<sup id="a1">[1](#f1)</sup> system to identify the  human activities for [Kinetics-400 dataset](https://deepmind.com/research/open-source/kinetics). The models use the Video Transformer approach with ResNet34 encoder<sup id="a2">[2](#f2)</sup>. We will see in this notebook how to create the following pipeline:

Reviewer: ```suggestion
This notebook demonstrates live human action recognition with OpenVINO. We use the [Action Recognition Models](https://docs.openvino.ai/latest/omz_models_group_intel.html#action-recognition-models) from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo), specifically the Encoder and Decoder from [action-recognition-0001](https://docs.openvino.ai/latest/omz_models_model_action_recognition_0001.html) . Both models create a sequence to sequence (`"seq2seq"`)<sup id="a1">[1](#f1)</sup> system to identify the  human activities for [Kinetics-400 dataset](https://deepmind.com/research/open-source/kinetics). The models use the Video Transformer approach with ResNet34 encoder<sup id="a2">[2](#f2)</sup>. In the notebook we show how to create the following pipeline:
```

Author: @ryanloney Do we have plans to update every notebooks for the latest version of OpenVINO? Sorry for my ignorance.

Reviewer: We plan to keep the notebooks working with the latest version of OpenVINO. We only introduce API changes in 202X.1 releases, so very infrequently we will need changes. Right now we are adapting the existing notebooks to work with 2022.1, coming out very soon.

<p align="center" width="100%">
    <img width="80%" src="https://user-images.githubusercontent.com/10940214/148401661-477aebcd-f2d0-4771-b107-4b37f94d0b1e.jpeg"> 
</p>

> At the bottom of this notebook, you will see live inference results from your webcam. You can also upload a video file.

Reviewer: ```suggestion
At the end of this notebook, you will see live inference results from your webcam. You can also upload a video file.
```
